packages/app/src/specs/SpecsList.vue

```diff
@@ -170,7 +171,7 @@
               />
               <div
                 v-else-if="row.data.isLeaf && row.data.data?.cloudSpec?.fetchingStatus === 'FETCHING'"
-                class="bg-gray-50 rounded-[20px] w-full animate-pulse"
+                class="bg-gray-50 rounded-[20px] w-80px animate-pulse"
```

Comment:
what I would love to do here is do full width unless it would overlap with the previous column, and then do the max that doesn't overlap, but I did this for simplicity - still debating if it makes it better or worse

Reply:
Here's an example of what I mean:

<img width="1124" alt="Screen Shot 2022-06-12 at 9 37 44 PM" src="https://user-images.githubusercontent.com/7063516/173264485-857592a8-00e4-4cfc-9af2-49b4ea0e601f.png">

Reply:
It seems to be correctly reflecting the 1:2 ratio for Last updated vs. Latest runs, keeping in mind Last updated is left aligned and Latest runs is right aligned. I believe the overlapping is the text in Last updated overflowing. I added background colors to these columns in the screenshot below to visualize it. I think we need to handle the text overflow for the first 2 columns. 
![image](https://user-images.githubusercontent.com/15657366/173267171-ab0787e2-c79f-4b3e-94ff-cd97c159efc6.png)

Reply:
I was PR-surfing when I found this PR seemingly addressing the text wrapping issue #22226